src/toolkits/object_detection/one_shot_object_detection/one_shot_object_detector.cpp

```diff
@@ -157,47 +195,19 @@ gl_sframe augment_data(const gl_sframe &data,
     size_t row_number=segment_start;
     for (const auto& background_ft: backgrounds.range_iterator(segment_start, segment_end)) {
       row_number++;
+      flex_image flex_background = background_ft.to<flex_image>();
```

Comment:
Could we just use `turi::image_util::decode_image` to make this one, more readable, line:
```
flexible_type flex_background = decode_image(background_ft.to<flex_image());
```
?

Reply:
Done.